ska_builder.py

```diff
@@ -102,16 +109,25 @@ def get_repo(name, tag):
 def build_package(name):
     print("Building package %s." % name)
     pkg_path = os.path.join(pkg_defs_path, name)
-    cmd_list = ["conda", "build", pkg_path, "--croot",
-                BUILD_DIR, "--no-test", "--old-build-string",
-                "--no-anaconda-upload", "--skip-existing",
+    cmd_list = ["conda", "build", pkg_path,
+                "--croot", BUILD_DIR,
+                "--old-build-string",
+                "--python", "3.6",
+                "--no-anaconda-upload",
                 "--numpy", NUMPY,
-                "--perl", PERL]
-    subprocess.run(cmd_list, check=True)
+                "--perl", PERL
+                ]
+    if not args.test:
+        cmd_list.append("--no-test")
+    if not args.force:
+        cmd_list.append("--skip-existing")
+
+    print('Running:\n' + ' '.join(cmd_list) + '\n')
+    subprocess.run(cmd_list, check=True, shell=True)
```

Comment:
The addition of shell=True seems to break this for me on linux.  The conda build command just moves to spitting out its helptext and I'm not quite sure why.

Reply:
Sorry, yes I noticed that as well, so this needs to be conditional on Windows (`shell=<is_windows>`).